docs/Documentation/Applications/gaussian.md

```diff
@@ -0,0 +1,122 @@
+# Running Gaussian16 Software Jobs
+
+*Learn about the Gaussian16 electronic structure program and how to run Gaussian16 jobs at NREL.*
+
+**Important Note:** To run Gaussian16, users must be a member of the Gaussian user group. To be added to the group, contact [HPC-Help](mailto:hpc-help@nrel.gov). In your email message, include your username and copy the following text agreeing not to compete with Gaussian, Inc.:
```

Comment:
I think this could go in an "important" admonition box like you used in the GAMS page on your other PR.